api/envoy/api/v2/core/health_check.proto

```diff
@@ -63,6 +63,12 @@ message HealthCheck {
 
   // [#comment:next free field: 10]
   message HttpHealthCheck {
+    enum CodecClientType {
+      Http1 = 0;
```

Comment:
Enum values should be ALL_CAPS. Also, the default should either be UNSET or something that you definitely are confident makes sense as the default for all time going forward.